livre/eq/EventHandler.cpp

```diff
@@ -274,6 +274,13 @@ template< class C > void EventHandler< C >::init()
 }
 
 template< class C >
+bool EventHandler< C >::handleEvent( eq::EventType, const eq::Event& )
+{
+    _impl->config.postRedraw();
```

Comment:
These are events handled here, and I think all should redraw:
    case EVENT_WINDOW_CLOSE:
    case EVENT_WINDOW_HIDE:
    case EVENT_WINDOW_EXPOSE:
    case EVENT_EXIT:
    case EVENT_WINDOW_POINTER_GRAB:
    case EVENT_WINDOW_POINTER_UNGRAB: